docs/source/getting_started/index.ipynb

```diff
@@ -1139,6 +1159,54 @@
     "tbl.table.head(2)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "762799a5-5b99-46ca-8b68-401dfdf10100",
+   "metadata": {},
+   "source": [
+    "To include multiple years of data, you can include the years in the \"year\" parameter in the form of [Start Year, Stop Year]\n",
```

Comment:
A couple minor suggestions:
1. Maybe "To request" instead of "To include"? 
2. Date ranges can only be used for multi-year datasets. I suggest adding something to indicate that.

Reply:
Got it! I'll reword the statement.